code/modules/hydroponics/seeds.dm

```diff
@@ -1513,3 +1533,79 @@
 	water_consumption = 6
 	nutrient_consumption = 0.10
 	growth_stages = 4
+
+// Vox Food
+
+/datum/seed/mushroom/chicken
+	name = "chickenshroom"
+	seed_name = "chickenshroom"
+	display_name = "chicken of the stars"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/chickenshroom)
+	mutants = null
+	packet_icon = "mycelium-chickenshroom"
+	plant_dmi = 'icons/obj/hydroponics2.dmi'
+	plant_icon = "chickenshroom"
+	chems = list("nutriment" = list(2,10))
+	consume_gasses = list("nitrogen"=20) //Really likes its nitrogen. Planting on main station may mess with room air mix.
+
+	lifespan = 30
+	growth_stages = 3
+	maturation = 4
+	yield = 3
+	potency = 0
+
+/datum/seed/apple/wood
+	name = "woodapple"
+	seed_name = "woodapple"
+	display_name = "woodapple tree"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/woodapple)
+	packet_icon = "seed-woodapple"
+	plant_dmi = 'icons/obj/hydroponics2.dmi'
+	plant_icon = "woodapple"
+	chems = list("sugar" = list(1,10))
+
+	growth_stages = 3
+	maturation = 4
+	yield = 4
+	potency = 20
+	ligneous = 1
+
```

Comment:
Well, had an ahelp about somebody finding a wood apple seed that had 0 production time

> [09:02:52] HELP: Jubisloviu/(Charles Vauban) (?) (PP) (VV) (SM) (JMP) (CA) : quick question, is the woodapple from vox hydroponics supposed to have 0 production time? 

Also, applying a plant analyzer to a vapor sac makes it explode. This kills the research botanist